lib/cluster.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+/**
+ * In cluster masters, extends Registry class with a `clusterMetrics` method,
+ * which returns aggregated metrics for all workers.
+ *
+ * In cluster workers, listens for and responds to requests for metrics by the
+ * cluster master.
+ */
+
+const cluster = require('cluster');
+const Registry = require('./registry');
+const AGGREGATORS = require('./metricAggregators').AGGREGATORS;
+
+const GET_METRICS_REQ = 'prom-client:getMetricsReq';
```

Comment:
Can this be a `Symbol` instead of a string? Not sure how serialization works here

Reply:
Not as far as I know, without using `Symbol.for(string)` (which I don't think is any better than just using the string).

Reply:
Yeah, just the string is fine then 🙂